usmqe_tests/conftest.py

```diff
@@ -311,3 +311,44 @@ def fill_memory():
             raise OSError(stderr)
         return request.param
     return measure_operation(fill_memory)
+
+
+@pytest.fixture(params=[70, 95])
+def workload_swap_utilization(request):
+    """
+    Returns:
+        dict: contains information about `start` and `stop` time of stress-ng
+            command and its `result`
+    """
+    def fill_memory():
+        """
+        Use `stress-ng` tool to stress swap memory for 4 minutes to given
+        percentage
+        """
+        run_time = 240
+        SSH = usmqe.usmssh.get_ssh()
+        host = pytest.config.getini("usm_cluster_member")
+
+        meminfo_cmd = "free | awk '{if (NR!=1) print $2}'"
+        _, stdout, _ = SSH[host].run(meminfo_cmd)
+        mem_total, swap_total, _ = stdout.decode("utf-8").split("\n")
```

Comment:
I changed it so that there is always second column of second and third row. If swap is setup then it should work but I am not sure with other configuration. I also added comment describing what is the `meminfo_cmd` used for.

Reply:
I was asking mainly because of output looks like this:
```
# free
             total       used       free     shared    buffers     cached
Mem:       8191220    5528148    2663072        176     444668    1713068
-/+ buffers/cache:    3370412    4820808
Swap:       524284          0     524284
```

But I'm not sure, if it is worth to consider it as it seems to be valid for RHEL 6/CentOS 6 (older versions of `procps`).

Reply:
Or you can go to ` /proc/meminfo` directly.